src/olympia/blocklist/models.py

```diff
@@ -155,14 +156,24 @@ def get_blocks_from_guids(cls, guids):
         return blocks
 
 
+class BlockType(Enum):
+    BLOCKED = 'blocked'
+    SOFT_BLOCKED = 'soft_blocked'
+
```

Comment:
use `Choices` (see use elsewhere in the repo) to get some extra utility functions around this

Reply:
may as well switch `soft` to be a small int too

Reply:
That was my original idea 😆 so we would rather have choices with int value? @diox if you agree I'll update in this PR.

Reply:
Kept Boolean field but added the BlockType as a choices.